src/main/app/src/app/informatie-objecten/informatie-objecten.service.ts

```diff
@@ -130,12 +181,41 @@ export class InformatieObjectenService {
   }
 
   updateEnkelvoudigInformatieobject(
-    documentNieuweVersieGegevens: EnkelvoudigInformatieObjectVersieGegevens,
+    uuid: string,
+    zaakUuid: string,
+    infoObject: EnkelvoudigInformatieObjectVersieGegevens,
   ): Observable<EnkelvoudigInformatieobject> {
+    const formData = new FormData();
+    formData.append("uuid", uuid);
+    formData.append("zaakUuid", zaakUuid);
+    formData.append("titel", infoObject.titel);
+    formData.append(
+      "vertrouwelijkheidaanduiding",
+      infoObject.vertrouwelijkheidaanduiding,
+    );
+
+    formData.append("auteur", infoObject.auteur);
+    formData.append("status", infoObject.status);
+    formData.append("taal", JSON.stringify(infoObject.taal));
+    formData.append("bestandsnaam", infoObject.bestandsnaam);
+    formData.append("formaat", infoObject.formaat);
+    formData.append("file", infoObject.file, infoObject.bestandsnaam);
+    formData.append("beschrijving", infoObject.beschrijving);
+    if (infoObject.verzenddatum)
+      formData.append("verzenddatum", infoObject.verzenddatum);
+    if (infoObject.ontvangstdatum)
+      formData.append("ontvangstdatum", infoObject.ontvangstdatum);
+    formData.append("toelichting", infoObject.toelichting);
```

Comment:
Do we need to check all of these for existence (null, undefined) ? 

We can have a helper function that takes one string argument and:
1. checks if infoObject[key] is null
2. if not null adds infoObject[key] under key, or
3. applies the passed function (JSON.stringify) on the value